### DISCUSSION TITLE
The Annotated Turing  [LIDO] ✅

### DISCUSSION THREAD

**Discussion by lincolixavier** [pilhacheia/clube-do-livro-dev#8] [Books]
The Annotated Turing: A Guided Tour Through Alan Turing's Historic Paper on Computability and the Turing Machine

Programming Legend Charles Petzold unlocks the secrets of the extraordinary and prescient 1936 paper by Alan M. Turing
Mathematician Alan Turing invented an imaginary computer known as the Turing Machine; in an age before computers, he explored the concept of what it meant to be computable, creating the field of computability theory in the process, a foundation of present-day computer programming.

[Link Amazon](https://www.amazon.com.br/gp/product/0470229055?ie=UTF8&tag=lincolixavier-20&camp=1789&linkCode=xm2&creativeASIN=0470229055)

**Comment by gustavoferrazfontes** [comment]
I never heard about that book. Seems good! I would like to read that!

**Comment by lincolixavier** [comment]
Its a legendary book, unfortunately there is only an english version =/
